docs/contents/components/forms/slider/accessibility.ja.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Slider
+description: "`Slider`は、ユーザーが値の範囲から選択するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/slider"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`Slider`は、アクセシビリティに関して[WAI-ARIA - Slider Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/slider/)に従います。
+
+例えば曜日が数値で表されているような、`aria-valuenow`の値がユーザーフレンドリーでない場合、`aria-valuetext`に`"Monday"`等の人間が理解可能な文字列を設定すると、スクリーンリーダーに読み上げられます。
+
+```tsx live=false
+const weekday = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+]
+
+const [value, onChange] = useState<number>(0)
+
+return (
+  <Slider
+    aria-valuetext={weekday[value]}
+    max={6}
+    min={0}
+    value={value}
+    onChange={onChange}
+  />
+)
+```
+
+## キーボード操作
+
+| キー         | 説明                                     | 状態 |
+| ------------ | ---------------------------------------- | ---- |
+| `Tab`        | つまみをフォーカスします。               | -    |
+| `ArrowRight` | 値を1ステップ増やします。 　             | -    |
+| `ArrowLeft`  | 値を1ステップ減らします。                | -    |
+| `ArrowUp`    | 値を1ステップ増やします。                | -    |
+| `ArrowDown`  | 値を1ステップ減らします。                | -    |
```

Comment:
It might be better to combine it like the documentation for the Rating component.
https://yamada-ui.com/ja/components/forms/rating/accessibility

```suggestion
| `ArrowRight`, `ArrowUp` | 値を1ステップ増やします。 　             | -    |
| `ArrowLeft`, `ArrowDown`   | 値を1ステップ減らします。                | -    |
```